app/models/tube/purpose.rb

```diff
@@ -67,5 +67,6 @@ def set_default_printer_type
 require_dependency 'qcable_tube_purpose'
 require_dependency 'illumina_htp/mx_tube_purpose'
 require_dependency 'illumina_htp/stock_tube_purpose'
+require_dependency 'illumina_htp/initial_stock_tube_purpose'
```

Comment:
Just wondering what this is for?

Reply:
`bundle exec rails db:seed` and `bundle exec rails record_loader:all` both fail without it.  The classes are lazy loaded during those processes and without declaring this dependency, they fail when they try to load the tube purpose with that type:

```
Lib Pool:
  type: IlluminaHtp::InitialStockTubePurpose
  target_type: StockMultiplexedLibraryTube
  stock_plate: false
```

Reply:
Ah sounds like a good bug fix 👍